server/project/project.go

```diff
@@ -412,7 +412,8 @@ func (s *Server) Update(ctx context.Context, q *project.ProjectUpdateRequest) (*
 		destCluster, err := argo.GetDestinationCluster(ctx, a.Spec.Destination, s.db)
 		if err != nil {
 			if err.Error() != argo.ErrDestinationMissing {
```

Comment:
I really wish we could use `errors.Is`. Should we still increment the `invalidDstCount++` before continuing?